SNIP-23.md

```diff
@@ -0,0 +1,41 @@
+# SNIP-23 - Improved UX to [SNIP-20](/SNIP-20.md) Send Operations
+
+This document describes an optional enhancement to the SNIP-20 spec. Contracts that support the existing SNIP-20 standard are still considered compliant, and clients should be written to benefit from these features when available, but provide fallback for when these features are not available.
+
+The feature specified in this document is an improved UX for the `Send` and `SendFrom` operations, aimed at enriching SNIP20 tokens composability and usage.
+
+- [SNIP-23 - Improved UX to SNIP-20 Send Operations](#snip-23---improved-ux-to-snip-20-send-operations)
+  - [The optional `recipient_code_hash` parameter](#the-optional-recipient_code_hash-parameter)
+    - [Updated `Send` interface](#updated-send-interface)
+    - [Updated `SendFrom` interface](#updated-sendfrom-interface)
```

Comment:
I'd remove the link back to the head of the page and leave just the other three links with the indentation you used.

Reply:
It's how my .md extension does table of contents. :shrug:

Reply:
Sure, but does it also force you to keep the ToC this way?
It's just a stylistic choice by me - people know what document they're on so having the main heading in the ToC is redundant

Reply:
Done